tests/.gitignore

```diff
@@ -0,0 +1,2 @@
+*.pyc
+
```

Comment:
Should not be necessary. It's already in the root .gitignore.

Reply:
OK.

Reply:
I've found reference, which states that, after they tested this scenario (does a root `.gitignore` pass its ignore to subdirectory), and their test results conclude that `.gitignore` files are not inherited by subdir!  https://stackoverflow.com/a/20880582/4339193

Reply:
No, this line is not necessary.

test is not a nested git repo and the author of the comment you link to is really doing weird things with its own configuration.